woocommerce/woocommerce-android
# improve validation error msg when addin coupons in order creation

Partially Closes: #9314


### Description


This PR adds a more explicit error message to the coupon text field in Order Creation for invalid coupon codes

### Testing instructions


1. Go to the orders tab
2. click on + to add a new order
3. add a product
4. click on Add coupon
5. type any random thing that is not a valid coupon code in the text field
6. click on Done
7. make sure the error message is displayed

### Images/gif


| Before | After |
| ---- | ---- |
| <img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/7b223298-2e6b-4ffa-a992-4329fcde278b" width="350"/> | <img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/ac3643b8-267e-49b3-98a4-f2ede9ceafd0" width="350"/> |
